.github/workflows/dotnet.yml

```
@@ -0,0 +1,56 @@
+name: Chorus .NET6
+
+on:
+  [push, pull_request, workflow_dispatch]
+
+jobs:
+  build-and-test:
+    env:
+      NUGET_AUTH_TOKEN: ${{secrets.GITHUB_TOKEN}}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+    
+    steps:
+    - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # v3.0.2
+      with:
+        fetch-depth: 0 # fetch full history for GitVersion
+    - name: Setup .NET
+      uses: actions/setup-dotnet@c0d4ad69d8bd405d234f1c9166d383b7a4f69ed8 # v2.1.0
+      with:
+        dotnet-version: 6.0.x
+        source-url: https://nuget.pkg.github.com/sillsdev/index.json
+    - name: Fix Python on Linux (for mercurial)
+      if: matrix.os == 'ubuntu-latest'
+      run: sudo apt install python-is-python2
+    - name: Restore
+      run: dotnet restore
+    - name: Build
+      run: dotnet build --no-restore -c Release
+    - name: Test
+      run: dotnet test
```

A comment would be nice why we can't use `--no-build -c Release`.

```suggestion
    - name: Test
      # it would be nice to run tests with `--no-build -c Release` but that causes test
      # failures on Ubuntu where it suddenly can't open the display.
      run: dotnet test
```